pkg/utils/mount.go

```diff
@@ -50,7 +50,19 @@ func CheckMountReadyAndSubPathExist(fluidPath string, mountType string, subPath
 	glog.Infoln(command)
 	stdoutStderr, err := command.CombinedOutput()
 	glog.Infoln(string(stdoutStderr))
-	return err
+
+	if err != nil {
+		// exitcode=1 indicates timeout waiting for mount point to be ready
+		if strings.HasPrefix(err.Error(), "exit status 1") {
+			return errors.New("timeout waiting for FUSE mount point to be ready")
+		}
+		// exitcode=2 indicates subPath not exists
+		if strings.HasPrefix(err.Error(), "exit status 2") {
+			return fmt.Errorf("subPath \"%s\" not exists under FUSE mount", subPath)
+		}
+		return err
+	}
+	return nil
 }
 
```

Comment:
How about using  `errors.As` instead of `strings.HasPrefix`?

```golang
if errors.Is(err, exec.ErrNotFound) {
    return errors.New("fusemount command not found")
}
var fuseMountErr *exec.ExitError
if errors.As(err, &fuseMountErr) {
    switch fuseMountErr.ExitCode() {
    case 1:
        return errors.New("timeout waiting for FUSE mount point to be ready")
    case 2:
        return fmt.Errorf("subPath \"%s\" not exists under FUSE mount", subPath)
    }
}
return err
```